rfcs/0001-meta-schedule-autotensorir.md

```diff
@@ -0,0 +1,297 @@
+* Feature Name: Meta Schedule (AutoTensorIR)
```

Comment:
Can we drop the AutoTensorIr name? We can just call all of this Meta Schedule.

Reply:
probably we could rename AutoTensorIR to AutoTIR, but I thought for quite a while and finally decided to add this name - the rationale is that we have been advocating "AutoTIR" as the code name for half a year, so it might help people who are searching for this term. In the meantime, it is ensured that AutoTIR doesn't appear in the body of the document, so that we can have more consistency

Reply:
If so, maybe we should alias it to `AutoTIR` so that people could search it in code.

Reply:
I thought a bit. Let's only use "AutoTIR" once in the title, and in other places let's be consistent with the name "meta schedule"